build.sc

```diff
@@ -1,37 +1,76 @@
 import coursier.Repository
 import coursier.maven.MavenRepository
 import mill._
-import scalalib._
-import scalafmt._
+import mill.scalalib._
+import mill.scalalib.scalafmt._
 
-trait ScalaTest extends TestModule {
-  private val ScalatestVersion = "3.0.5"
-  override def ivyDeps = Agg(ivy"org.scalatest::scalatest::$ScalatestVersion".excludeOrg("org.junit"))
-  def testFrameworks = Seq("org.scalatest.tools.Framework")
-}
+/** Version of Clio to build / test against. */
+private val ClioVersion = "9dd03079781f6855a8e4582550fbdacd1433f4d2"
```

Comment:
It seems like we'd want this to be an input that jenkins will provide, with a default like `develop`. Would that be hard to do?

Reply:
will do